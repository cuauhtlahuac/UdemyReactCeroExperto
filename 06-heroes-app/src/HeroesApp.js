import React from 'react';

import AppRouter from 'routers/AppRouter';

const HeroesApp = () => <AppRouter />;

export default HeroesApp;
