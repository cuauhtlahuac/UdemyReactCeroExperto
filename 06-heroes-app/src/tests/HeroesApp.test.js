import React from 'react';
import { shallow } from 'enzyme';

import HeroesApp from 'HeroesApp';

describe('HeroesApp tests', () => {
	test('should render correctly', () => {
    const component = shallow(<HeroesApp />);
    
    expect(component).toMatchSnapshot();
	});
});
