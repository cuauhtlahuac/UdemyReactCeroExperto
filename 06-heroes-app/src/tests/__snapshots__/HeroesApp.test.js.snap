// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroesApp tests should render correctly 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
